src/App.vue

```diff
@@ -1,6 +1,7 @@
 <template>
 <div class="vue-world-map">
-  <Map />
+  <div :v-if="showColorBar" id="color_bar"></div>
```

Comment:
```suggestion
  <div v-if="showColorBar" id="color_bar"></div>
```
Mistake here `:v-if` instead of `v-if` causes showColorBar to not work

Reply:
good catch man,
thanks for the heads up